guides/common/modules/con_managing-dns-using-smart-proxies.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_DNS_Using_Smart_Proxies_{context}"]
+= Managing DNS Using {SmartProxies}
+
+{Project} can manage DNS records using your {SmartProxy}.
+DNS management contains updating and removing DNS records from existing DNS zones.
+Your {SmartProxy} can use multiple providers, allowing you to integrate with your existing DNS infrastructure or deploy a new one.
+
+After you have enabled DNS, your {SmartProxy} can manipulate any DNS server that complies with RFC 2136.
+
+.Available DNS Providers
+ifdef::orcharhino[]
+* `dns_dnscmd` - static DNS records in Microsoft Active Directory
+endif::[]
+* `dns_infoblox` - For more information, see {ProvisioningDocURL}Using_Infoblox_as_DHCP_and_DNS_Providers_provisioning[Using Infoblox as DHCP and DNS Providers] in _{ProvisioningDocTitle}_.
+ifndef::satellite[]
+* `dns_libvirt` - dnsmasq DNS via libvirt API
+endif::[]
+* `dns_nsupdate` - dynamic DNS update using nsupdate
+* `dns_nsupdate_gss` - dynamic DNS update with GSS-TSIG
```

Comment:
This provider is documented in `guides/common/modules/proc_configuring-dynamic-dns-update-with-gss-tsig-authentication.adoc`. Would be good to link back.

Reply:
Added a link.